### PR TITLE
IBM-28: KeycloakServiceAccount Link added to footer

### PR DESCRIPTION
**Pull request by cksanders**
https://jira.digital.homeoffice.gov.uk/browse/IBM-28

### CODE REVIEW

File: app/styles/main.scss [hunk 1]
```diff
@@ -63,6 +72,12 @@ header {
   }
 }
 
+footer {
+  margin-left: 10px;
```
Comment (chrisns) [inline app/styles/main.scss]: $left_margin

File: app/styles/main.scss [hunk 2]
```diff
@@ -63,6 +72,12 @@ header {
...
+  border-top: 1px solid $color_celeste_approx;
+  padding: 10px 0 10px
```
Comment (chrisns) [inline app/styles/main.scss]: padding: 10px 0; would be the same

File: app/styles/main.scss [hunk 3]
```diff
@@ -24,6 +24,15 @@ body {
   color: $color_mine_shaft_approx;
   margin: 0;
   font-size: $base_font_size;
+  a {
+    text-decoration: none;
+    cursor: pointer;
+    color: $blue;
+    margin-top: -$base_font_size;
```
Comment (chrisns) [inline app/styles/main.scss]: why do you need a negative margin?

File: app/styles/main.scss [hunk 4]
```diff
@@ -24,6 +24,15 @@ body {
   color: $color_mine_shaft_approx;
   margin: 0;
   font-size: $base_font_size;
+  a {
+    text-decoration: none;
+    cursor: pointer;
```
Comment (chrisns) [inline app/styles/main.scss]: its a link, you shouldn't need to tell it to have a pointer

File: app/views/home.html [hunk 5]
```diff
@@ -18,3 +18,6 @@
       cid="centre.id">
   </li>
 </ul>
+<footer>
+  <a href="{{keycloakLinkPath}}" ng-bind="keycloakLinkText"></a>
```
Comment (chrisns) [inline app/views/home.html]: should use ```ng-href```

File: app/styles/main.scss (hunk 4 above)
Comment (cksanders) [inline app/styles/main.scss]: this style was previously implemented (by you?) ... I did not intend to refactor existing styling decisions. However, I did refactor the code by separating 'a href' styling away from the class '.detail-toggle'

**Pull request closed without merging**